website/content/en/highlights/2021-11-16-implicit-namespacing.md

```diff
@@ -0,0 +1,70 @@
+---
+date: "2021-11-16"
+title: "Automatic namespacing for config files"
+description: "A guide that addresses the new automatic namespacing functionality"
+authors: ["barieom"]
+pr_numbers: []
+release: "0.18.0"
+hide_on_release_notes: false
+badges:
+  type: new feature
+---
+
+# Automatic namespacing for configuration
```

Comment:
These headings aren't needed, as the HTML template displays the title from the page metadata